meson_options.txt

```diff
@@ -340,6 +340,9 @@ Using CUSE, the device files are provided.
 option('linux-getrandom', type: 'feature', value: 'enabled',
        description: 'Enable the Linux getrandom system call support.')
 
+option('botan-rng', type: 'feature', value: 'disabled',
+       description: 'Enable the Botan >= 3 RNG support.')
```

Comment:
May I ask that you add a hint that this is a frontend, i.e. to supply randomness to Botan? See now the added backends and I would like to avoid confusion about the support.